src/misc/ontime.ts

```diff
@@ -0,0 +1,156 @@
+import { Bukkit, Statistic } from 'org.bukkit';
+import { CommandSender } from 'org.bukkit.command';
+import { Player } from 'org.bukkit.entity';
+import { PlayerQuitEvent } from 'org.bukkit.event.player';
+import * as yup from 'yup';
+import { DatabaseEntry } from '../common/datas/database';
+import { dataType } from '../common/datas/holder';
+import { dataView } from '../common/datas/view';
+
+const OntimeDatabaseEntry = new DatabaseEntry('ontime', 'ontime-key');
+const OntimeData = dataType('ontimeData', {
+  list: yup
+    .array(
+      yup
+        .object({
+          uuid: yup.string().required(),
+          name: yup.string().required(),
```

Comment:
Player name should be looked up based on UUID, not saved - it might change.

Reply:
Does this cause performance issues if we get the name with UUID? Does this make a request to Mojang's servers? The name is updated every time the player visits the server, and I think it might be slow to request 100 names every time someone runs ontime-command.